File: Problem_set_1/Introductory_problems/p05_Permutation/Permutations.py
number = int(input())

even_num = []
odd_num = []

if(number > 3):
    for i in range(1, number + 1):
        if(i % 2 == 0):
            even_num.append(str(i))
        else:
            odd_num.append(str(i))
        
    print(" ".join(even_num + odd_num))
elif number == 1:
    print(number)
else:
    print("NO SOLUTION")

File: Problem_set_1/Introductory_problems/p05_Permutation/Permutations.ts
const input = require("prompt-sync")();

let num: number = Number(input())

let even_num: string[] = [];
let odd_num: string[]= [];

if(num > 3){
    for(let i: number = 1; i <= num; i++){
        if(i % 2 == 0){
            even_num.push(String(i));
        } else{
            odd_num.push(String(i));
        }
    }
    console.log([...even_num, ...odd_num]. join(" "));
}else if(num == 1){
    console.log(num);
}else{
    console.log("NO SOLUTION");
}